packages/pwa/app/components/_app/index.jsx

```diff
@@ -102,15 +114,15 @@ const App = (props) => {
 
     const onLogoClick = () => {
         // Goto the home page.
-        const path = homeUrlBuilder(HOME_HREF, targetLocale)
+        const path = homeUrlBuilder(HOME_HREF, {locale: targetLocale, site}, {site})
```

Comment:
```suggestion
        const path = homeUrlBuilder(HOME_HREF, {locale: targetLocale, site})
```
The 3rd parameter is not needed. `homeUrlBuilder` accepts only 2 parameters.

Reply:
Good catch, thank you.